app/helpers/admin/courses_helper.rb

```diff
@@ -3,8 +3,13 @@ module Admin::CoursesHelper
   def full_admin_courses_url_with_params(*current_params, additional_param)
     admin_courses_url(request.params.slice(*current_params).merge(additional_param))
   end
-
+  
   def check_owner(course)
     current_user.id == course.creator_id
    end
+  
+   def get_creator(creator_id)
+     User.find_by(id: creator_id).username
```

Comment:
Xử lý nil chổ find_by nữa đi Kiều, chứ lỡ ra nil là lỗi ngay.

Reply:
Ok,thế nếu user tạo ra course mà bị xóa rồi thì t để username là none hỉ

Reply:
Chổ này t góp ý như thế này cho nó hợp lý hơn nhé. M khai báo trong model course cái quan hệ belongs to User thông qua creator_id (giả sử đặt tên là creator). Xong rồi chổ check_owner chỉ cần dùng current_user == course.creator là được, như thế hợp lý hơn và sau này dễ truy vấn hơn nếu cần thông tin creator nữa.

Reply:
ời t biết cái này rồi á,mà lúc trước kêu lấy mỗi username nên t định truy vấn như vậy mà không xuống model để thêm quan hệ vô á